### ISSUE TITLE
Strict parsing mode

### ISSUE THREAD

**Issue by josegonzalez** [m1/Env#3]
We should have a strict parsing mode for .env files, specifically around general variable assignment.

**Comment by m1** [comment]
Yeah I remember us discussing this months ago, do you remember the exact details?

**Comment by josegonzalez** [comment]
This: https://github.com/m1/Env#assignment

One of the reasons symfony wrote their own `.env` parser - lol - is because you have a loose interpretation of variable assignment.

**Comment by m1** [comment]
Is it just the loose variable assignment that needs to be changed for the strict mode?

**Comment by josegonzalez** [comment]
I'd imagine so, though there might be more.

**Comment by m1** [comment]
Okay cool, I'll try and do it this weekend :+1:
